lib/podcast_importer.rb

```diff
@@ -0,0 +1,267 @@
+# import script for a podcast live in feeder & dovetail
+require 'prx_access'
+require 'addressable/uri'
+require 'feedjira'
+require 'itunes_category_validator'
+
+class PodcastImporter
```

Comment:
So much similarity between this & PodcastImport -- also, both this and FeederImporter say `# import script for a podcast live in feeder & dovetail` at top. Wondering when each different one will be used?